service/policy/db/resource_mapping.go

```diff
@@ -5,15 +5,118 @@ import (
 	"log/slog"
 
 	sq "github.com/Masterminds/squirrel"
+	"github.com/google/uuid"
 	"github.com/jackc/pgx/v5"
+	"github.com/jackc/pgx/v5/pgtype"
 	"github.com/opentdf/platform/protocol/go/common"
 	"github.com/opentdf/platform/protocol/go/policy"
 	"github.com/opentdf/platform/protocol/go/policy/resourcemapping"
 	"github.com/opentdf/platform/service/logger"
 	"github.com/opentdf/platform/service/pkg/db"
+	"github.com/opentdf/platform/service/pkg/util"
 	"google.golang.org/protobuf/encoding/protojson"
 )
 
+/*
+ Resource Mapping CRUD
+
+ NOTE: uses sqlc instead of squirrel
+*/
+
+func (c PolicyDBClient) ListResourceMappingGroups(ctx context.Context, r *resourcemapping.ListResourceMappingGroupsRequest) ([]*policy.ResourceMappingGroup, error) {
+	list, err := c.Queries.ListResourceMappingGroups(ctx, r.GetNamespaceId())
+	if err != nil {
+		return nil, db.WrapIfKnownInvalidQueryErr(err)
+	}
+
+	resourceMappingGroups := make([]*policy.ResourceMappingGroup, len(list))
+
+	for i, rmGroup := range list {
+		resourceMappingGroups[i] = &policy.ResourceMappingGroup{
```

Comment:
Just curious if this intentionally did not include metadata labels/timestamps?

Reply:
Same question for GET below.

Reply:
Let's handle under a separate issue.

Reply:
Not intentional.  I think when I originally implemented this we did not have `created_at` or `updated_at` columns, nor did I fully understand what the metadata was used for at that time.

Reply:
https://github.com/opentdf/platform/issues/1398